src/Components/Pages/TransferPage.tsx

```diff
@@ -411,231 +497,263 @@ const TransferPage = () => {
+            <div className={classes.walletDesc}>
+              Move CERE tokens between
+              <br />
+              Ethereum and Cere Native Blockchain
             </div>
-          </section>
-        )}
-      </div>
-      {isReady &&
-        walletType === "Substrate" &&
-        accounts &&
-        accounts.length > 0 && (
-          <div>
-            <section className={classes.accountSelector}>
-              <SelectInput
-                label="Select account"
-                className={classes.generalInput}
-                options={accounts.map((acc, i) => ({
-                  label: acc.address,
-                  value: i,
-                }))}
-                onChange={(value) => selectAccount && selectAccount(value)}
-                value={accounts.findIndex((v) => v.address === address)}
-                placeholder="Select an account"
-              />
-            </section>
+            <Button
+              variant="primary"
+              className={classes.EthWalletBtn}
+              onClick={() => setWalletType("Ethereum")}
+            >
+              Connect with ERC20 Wallet
+            </Button>
+            <Button
+              variant="primary"
+              className={classes.SubsWalletBtn}
+              onClick={() => setWalletType("Substrate")}
+            >
+              Connect with Substrate Wallet
+            </Button>
           </div>
-        )}
-      <Formik
-        initialValues={{
-          tokenAmount: 0,
-          token: "",
-          receiver: "",
-        }}
-        validateOnChange={false}
-        validationSchema={transferSchema}
-        onSubmit={(values) => {
-          setPreflightDetails({
-            ...values,
-            tokenSymbol: tokens[values.token].symbol || "",
-          });
-          setPreflightModalOpen(true);
-        }}
-      >
-        {(props) => (
-          <Form
-            className={clsx(classes.formArea, {
-              disabled: !homeConfig || !address || props.isValidating,
-            })}
-          >
-            {console.log(destinationChainConfig)}
-            <section>
-              <SelectInput
-                label="Destination Network"
-                className={classes.generalInput}
-                disabled={!homeConfig}
-                options={destinationChains.map((dc) => ({
-                  label: dc.name,
-                  value: dc.chainId,
-                }))}
-                onChange={(value) => setDestinationChain(value)}
-                value={destinationChainConfig?.chainId}
-              />
+        ) : (
+          <>
+            <section className={classes.connected}>
+              <div>
+                <div className={classes.inputLabel}>Home Network</div>
+                <Typography
+                  className={classes.changeButton}
+                  variant="body1"
+                  onClick={async () => {
+                    await Promise.all([
```

Comment:
Have you reused this logic or implemented it from scratch?

Reply:
Implemented from scratch.